spec/auth_spec.rb

```diff
@@ -116,31 +116,105 @@
     end
 
     # see also exclusion_spec.rb
-    describe 'exclude' do
-      describe 'when a type other than Array provided' do
+    describe 'exclude list validation' do
+      describe 'when exclude list contains an array of valid strings' do
+        it 'does not raise any exception' do
+          args = { secret: secret, exclude: %w(/books /music) }
+          expect { Rack::JWT::Auth.new(inner_app, args) }.not_to raise_error
+        end
+      end
+
+      describe 'when exclude list contains an array of valid hashes' do
+        it 'does not raise any exception' do
+          args = {
+            secret: secret,
+            exclude: [
+              { path: '/books', methods: :all },
```

Comment:
If one sets `methods: [:all]` will it work?

Reply:
I have made the change to allow this to work.  So if you specify `[:all]`, it's equivalent to `:all`